tests/test_workspace.py

```diff
@@ -66,3 +66,12 @@ def test_remove_document(workspace):
     assert workspace.get_document(DOC_URI).source == DOC_TEXT
     workspace.remove_document(DOC_URI)
     assert workspace.get_document(DOC_URI)._source is None
+
```

Comment:
We are using 2 blank lines between functions.

Reply:
Not sure why my auto formatting didn't catch this.